packages/theme-chalk/src/button.scss

```diff
@@ -7,7 +7,7 @@
 @include b(button) {
   display: inline-block;
   line-height: 1;
-  height: 40px;
+  min-height: 40px;
```

Comment:
Did we have a sass variable for "40px"?

Reply:
There is `$--input-height: 40px !default;`